devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java

```diff
@@ -180,4 +199,20 @@ public void setLog(Log log) {
         MojoLogger.delegate = log;
     }
 
+    private void exportCustomManifestProperties(Set<String> propertiesToClear) {
+        for (Map.Entry<String, String> attribute : manifestEntries.entrySet()) {
+            String attributeKey = String.format("%s.\"%s\"", MANIFEST_ATTRIBUTES_PROPERTY_PREFIX, attribute.getKey());
+            System.setProperty(attributeKey, attribute.getValue());
```

Comment:
@glefloch we don't want to set System properties in mojos due to the possibility of concurrent builds.
The way to do this would be to move the Mojo parameters you added here to the QuarkusBootstrapMojo and the code propagating them as properties to the QuarkusBootstrapProvider.doBootstrap(...) in `effectiveProperties`.